src/Tribe/Venue.php

```diff
@@ -698,7 +698,9 @@ public static function get_fetch_callback( $event ) {
 		}
 
 		return static function () use ( $event ) {
-			$venue_ids = array_map( 'absint', (array) get_post_meta( $event, '_EventVenueID' ) );
+			$venue_ids = array_filter(
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 59f7057681611dae42dc63f33df94e3c</sub>